#!/usr/bin/env python
PACKAGE = "angle_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('costmap_critical',          int_t,    0, 'Critical cost for costmap cell',             128,  128,  255)
gen.add('orientation_critical',   double_t,    0, 'Critical cost for orientation restriction',   1.,    0,    1)
gen.add('orientation_coeff',      double_t,    0, 'Coefficient of orientation impact',         255.,    0, 1000)
gen.add('use_astar',                bool_t,    0, 'Use 16 neighbours instead of 8',            True)

exit(gen.generate(PACKAGE, PACKAGE, "AnglePlanner"))
